docs/contribute/cumulative-docs/badge-placement.md

```diff
@@ -0,0 +1,206 @@
+---
+navigation_title: Badge placement
+---
+
+% Audience: Technical writers and other frequent docs contributors
+% Goals:
+%   * Provide guidance on badge placement in specific situations
+
+# `applies_to` badge placement
+
+As you continue contributing to documentation and more versions are released,
+you might have questions about how to integrate `applies_to` badges in
+cumulative documentation.
+
+## General principles
+
+% Source: Brandon's PR review comment
+At a high level, you should follow these badge placement principles:
+
+* Place badges where they're most visible but least disruptive to reading flow
+* Consider scanning patterns - readers often scan for relevant information
+* Ensure badges don't break the natural flow of content
+* Use consistent placement patterns within similar content types
+
+## Specific elements
+
+There are more specific guidelines on badge placement to follow when using specific elements.
+
+### Headings
+
+Use [section annotations](/syntax/applies.md#section-annotations) immediately after a heading when the entire content between that heading and the next [heading](/syntax/headings.md) of the same or higher level is version or product-specific.
+
+For example, in the [Semantic text field type](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/semantic-text#custom-by-pipelines) page, all the content in this section is only applicable to Elastic Stack versions 9.0.0 and later.
+
+::::{image} ./images/heading-correct.png
+:screenshot:
+:alt: Correct use of applies_to with headings
+::::
+
+% FOR THE REVIEWER: IS THIS TRUE?
+% What do you think of allowing inline applies_to in headings as long as there is only one badge?
+:::{warning}
+Do **not** use [inline annotations](/syntax/applies.md#inline-annotations) with headings, which will cause rendering issues in _On this page_.
+
+::::{image} ./images/heading-incorrect.png
+:screenshot:
+:alt: Rendering error when using inline applies_to with headings
+::::
+:::
```

Comment:
We could fix this: https://github.com/elastic/docs-builder/pull/1661

I'm not sure I would document a bug here. If we fix it, we can just be prescriptive and say "Don't do this".